cmd/commands/dev.go

```diff
@@ -22,6 +22,7 @@ func NewCmdDev() *cobra.Command {
 	cmd.Flags().StringP("port", "p", "8288", "port to run the API on")
 	cmd.Flags().String("dir", ".", "directory to load functions from")
 	cmd.Flags().StringSliceP("sdk-url", "u", []string{}, "SDK URLs to load functions from")
+	cmd.Flags().Bool("docker", false, "enable docker-based steps")
```

Comment:
Do we need to alter any docs for this?

Reply:
This is legacy and isn't supported for `inngest dev`, so we shouldn't!